File: arrays/min-steps-in-infinite-grid.cpp
int Solution::coverPoints(vector<int> &A, vector<int> &B) {
    int ans = 0;
    int n = A.size();
    for(int i=0; i<n-1; i++){
        int x=abs(A[i+1]-A[i]);
        int y=abs(B[i+1]-B[i]);
        ans += max(x,y);
    }
    return ans;
}

File: arrays/min-steps-in-infinite-grid.py
class Solution:
    def coverPoints(self, A, B):
        dist = 0
        for i in range(len(A)-1):
            x = abs(A[i]-A[i+1])
            y = abs(B[i]-B[i+1])
            dist += max(x.y)
        return dist
